app/views/transactions/new.html.erb

```diff
@@ -1,13 +1,20 @@
-<nav class="header">
 
-   <h2>SMART SAVER</h2>
-  <div>
-    <p> NEW TRANSACTION </p>
-  </div>
-  <div>
-    <%= link_to "Back", new_transaction_path, class: "back" %>
-  </div>
+<nav class="bg-primary text-white d-flex justify-content-between p-4 mt-1">
+<div>
+<%= link_to new_transaction_path, class: "text-white fs-4 " do %>
+ <i class="fa fa-angle-left" aria-hidden="true"></i>
+<% end %>
+</div>
+
+<div>
+ <p class: "text-white fs-3 fw-bold "> NEW TRANSACTION  </p>
+</div>
 
+<div>
+<%= link_to root_path, class: "text-white fs-4 " do %>
+ <i class="fa fa-search" aria-hidden="true"></i>
+<% end %>
+</div>
 </nav>
 
```

Comment:
- [ ] KIndly ensure that on the Add new transaction page, the link back to the home page works, currently after clicking multiple times it does not work

Reply:
This change has been done well